path_service_principal_test.go

```diff
@@ -608,12 +637,12 @@ func TestRoleAssignmentWALRollback(t *testing.T) {
 			"azure_roles": fmt.Sprintf(`[
 			{
 				"role_name": "Storage Blob Data Owner",
-				"scope":  "/subscriptions/%s/resourceGroups/vault-azure-secrets-test1"
+				"scope":  "/subscriptions/%s/resourceGroups/%s"
 			},
 			{
 				"role_name": "Reader",
-				"scope":  "/subscriptions/%s/resourceGroups/vault-azure-secrets-test2"
-			}]`, subscriptionID, subscriptionID),
+				"scope":  "/subscriptions/%s/resourceGroups/%s"
+			}]`, subscriptionID, subscriptionID, resourceGroup, resourceGroup),
```

Comment:
Shouldn't this be `subscriptionID, resourceGroup, subscriptionID, resourceGroup`?

Reply:
Yes it should be. This broke the test. Fixed this.